src/neptune/versionHelper.d

```diff
@@ -0,0 +1,334 @@
+/*******************************************************************************
+
+    Helper to deal with version strings
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH. All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE.txt for details.
+
+*******************************************************************************/
+
+module neptune.versionHelper;
```

Comment:
Agreed, but I think all the things we note should be added as issues, otherwise they will get lost.